lib/models/sagemaker-schedule.ts

```diff
@@ -0,0 +1,48 @@
+import * as cdk from 'aws-cdk-lib';
+import * as scheduler from 'aws-cdk-lib/aws-scheduler';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as sagemaker from "aws-cdk-lib/aws-sagemaker";
+import { Construct } from 'constructs';
+
+export function createStartSchedule(scope: Construct, id: string, sagemakerEndpoint: sagemaker.CfnEndpoint, role: iam.Role, scheduleExpression: string) {
+  const startSchedule = new scheduler.CfnSchedule(scope, `${id}-${sagemakerEndpoint.endpointName}-startScheduler`, {
+    name: `StartScheduler-${id}-${sagemakerEndpoint.endpointName}`,
```

Comment:
With the ID in here, do you need that in the string? Is there a scenario where you end up with multiple start schedules for the same endpoint name?